applications/integrationtests/CMakeLists.txt

```diff
@@ -1,25 +1,14 @@
-if(ENABLE_GEOMETRYCONCRETE)
-    add_subdirectory(geometryConcrete)
-endif()
-
-if(ENABLE_METAMODEL)
-    add_subdirectory(metamodel)
-endif()
-
-if(ENABLE_OPTIMIZE)
-    add_subdirectory(optimize)
-endif()
+unset(all_integration_tests CACHE)
```

Comment:
@Psirus  is the expert here, but I'll try. The target `integrationtests` is basically a list of corresponding executables. In this case, this list is the variable `all_integration_tests`. In case you remove, rename or add new tests, this list has to be updated as well. For the target `unit` we are doing this technique for a while and several times I ran into problems. Especially if you delete a test, it will still be in the target list. I had to manually clear the variable in my CMakeCache.txt to fix that. I hope and guess that this `unset` command clears the variable automatically.

Reply:
Yes, unset is just for clearing that variable. Otherwise we would accumulate duplicate tests or try to run deleted tests, as @TTischer said.